policies/team-membership.md

```diff
@@ -44,7 +44,8 @@ Tenure schedule:
 |---------------|--------------|-------------|
 | Matt Stratton | 1 Nov 2020   | 31 Oct 2023 |
 | Yvo van Doorn | 1 Nov 2020   | 31 Oct 2022 |
-| TBD           | 1 Nov 2020   | 31 Oct 2024 |
+| Dan Maher     | 1 Nov 2022   | 31 Oct 2024 |
+| Katie McLaughlin | 1 Nov 2022 | 31 Oct 2025 |
```

Comment:
because it's rolling, we only want one co-chair "coming off" at a time, I think? so every year you have one coming off and an election for a replacement. 

I arbitrarily chose Katie as the one going longer, but we could swap the two of you

so in theory it goes like this:

2022 - yvo rolls off 
2023 - matt rolls off and is replaced
2024 - dan rolls off and is replaced
2025 - katie rolls off and is replaced
2026 - matt's replacement rolls off and is replaced
....

Reply:
Sure, but if it's a 
> tenure of two years

That is at odds with Katie's tenure, which is three years. To be clear, if Katie wants to do this job for a decade I'm OK with that—I'm just looking to understand how the proposed schedule lines up with the language employed to describe that schedule.

Reply:
yeah we had this same problem when yvo and I kicked off, which is why we had the language around that the first time we would have someone who did it for three years instead of two.

If we have both of you roll off in 2024 we just have to replace with two at the same time.

I guess if neither of you wants the three year gig, we can modify it for two years and just have a "two new people come in every other year" situation which ... isn't terrible?

Reply:
if we are going to have a three-year tenure for one of you, then we need to update this statement to include that `(exception is for the first co-chairs of the new schedule, Matt Stratton and Yvo van Doorn. In order to accommodate the staggering, Matt will serve a 3 year tenure.)`

Reply:
I'd probably modify it to be more generic. I can imagine a future situation where we elect 2 new folks at the same time again (e.g. someone needs to step down and and the planned chair that rolls off is unable to serve for an additional year). Generic text would be something like `Chairs will serve a tenure of two years, in a staggered tenure schedule. If multiple chairs are added at the same time, the tenure of one chair may be shortened to one year or extended to three years in order to maintain the staggering.`

Reply:
I updated this with the feedback from @jyee